package.json

```diff
@@ -7,7 +7,17 @@
     "@azure-tools/cadl-ranch-mockapis": "file:test/CadlRanchMockApis",
     "@microsoft.azure/autorest.testserver": "3.3.48",
     "autorest": "3.6.3",
-    "@typespec/openapi": "^0.49.0",
-    "@typespec/openapi3": "^0.49.0"
+    "@azure-tools/typespec-azure-resource-manager": "0.36.0",
+    "@azure-tools/typespec-client-generator-core": "0.36.0",
+    "@azure-tools/typespec-azure-core": "0.36.0",
```

Comment:
Tests in this repo import these packages, for example here:

https://github.com/Azure/autorest.csharp/blob/b5a406804e7659a9597ef1988c7dca5519c8c68e/test/TestProjects/FirstTest-TypeSpec/FirstTest-TypeSpec.tsp#L3

When calling `tsp compile` on these specs, these packages must be loadable by `npm`.  These packages are dependencies in `emitter.csharp/package.json`, so this may have worked previously (by accident), but `npm` doesn't guarantee that transitive dependencies are installed at the top level of `node_modules`.  The only way to guarantee these packages are loadable, is to have them in a `package.json` in a folder **above** the specs.

One other solution I just thought of, would be to move the TypeSpec tests under the `emitter.csharp` folder.  This should also work, if you run `tsp compile .` from the spec folder.

Reply:
This is because its the only way to force a package to be installed at the root level